app/views/layouts/blacklight.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for(:content) do %>
```

Comment:
Can you drop an ERB comment here saying why we decided to override this layout?

```
<%# We are overriding this Blacklight layout because blah blah %>
```